tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -347,6 +350,60 @@ void SetHtmlResponseContent()
             return blockingAction;
         }
 
+        private string GetJsonResponse()
+        {
+            if (_blockedJsonTemplateCache != null)
+            {
+                return _blockedJsonTemplateCache;
+            }
+
+            if (_settings.BlockedJsonTemplatePath != null && AppDomain.CurrentDomain.BaseDirectory != null)
+            {
+                var fullPath =
+                    Path.IsPathRooted(_settings.BlockedJsonTemplatePath)
+                        ? _settings.BlockedJsonTemplatePath
+                        : Path.Combine(AppDomain.CurrentDomain.BaseDirectory, _settings.BlockedJsonTemplatePath);
+                if (File.Exists(fullPath))
+                {
+                    _blockedJsonTemplateCache = File.ReadAllText(fullPath);
+                }
+            }
+
+            if (_blockedJsonTemplateCache == null)
+            {
+                _blockedJsonTemplateCache = SecurityConstants.BlockedJsonTemplate;
+            }
+
+            return _blockedJsonTemplateCache;
+        }
+
+        private string GetHtmlResponse()
+        {
+            if (_blockedHtmlTemplateCache != null)
+            {
+                return _blockedHtmlTemplateCache;
+            }
+
+            if (_settings.BlockedHtmlTemplatePath != null && AppDomain.CurrentDomain.BaseDirectory != null)
+            {
+                var fullPath =
+                    Path.IsPathRooted(_settings.BlockedJsonTemplatePath)
```

Comment:
Is this a bug, using BlockedJsonTemplatePath instead of BlockedHtmlTemplatePath?

Reply:
Yes, fixed in 18d660033